benchmarks/transport/mpi_p2p_bi_avail_mt.cpp

```diff
@@ -3,10 +3,38 @@
 #include <string.h>
 #include <atomic>
 
-#include <ghex/transport_layer/ucx/threads.hpp>
 #include <ghex/common/timer.hpp>
 #include "utils.hpp"
 
+#ifdef USE_OPENMP
+#include <omp.h>
+#define DO_PRAGMA(x) _Pragma(#x)
+
+#define GET_THREAD_NUM()  omp_get_thread_num()
```

Comment:
This seems to be only valid for OpenMP. Is this test only available for this, or should the code use thread-primitives?

Reply:
this is only for this benchmark, no need to make it general.